web_src/js/features/projects.js

```diff
@@ -0,0 +1,97 @@
+const {csrf} = window.config;
+
+export default async function initProject() {
+  if (!window.config || !window.config.PageIsProjects) {
+    return;
+  }
+
+  const {Sortable} = await import(/* webpackChunkName: "sortable" */'sortablejs');
+  const boardColumns = document.getElementsByClassName('board-column');
+
+  for (const column of boardColumns) {
+    new Sortable(
+      column.getElementsByClassName('board')[0],
+      {
+        group: 'shared',
+        animation: 150,
+        onAdd: (e) => {
+          $.ajax(`${e.to.dataset.url}/${e.item.dataset.issue}`, {
+            headers: {
+              'X-Csrf-Token': csrf,
+              'X-Remote': true,
+            },
+            contentType: 'application/json',
+            type: 'POST',
+            error: () => {
+              e.from.insertBefore(e.item, e.from.children[e.oldIndex]);
+            },
+          });
+        },
+      }
+    );
+  }
+
+  $('.edit-project-board').each(function () {
+    const projectTitle = $(this).find(
+      '.content > .form > .field > .project-board-title'
+    );
+
+    $(this)
+      .find('.content > .form > .actions > .red')
+      .on('click', function (e) {
+        e.preventDefault();
+
+        $.ajax({
+          url: $(this).data('url'),
+          data: JSON.stringify({title: projectTitle.val()}),
+          headers: {
+            'X-Csrf-Token': csrf,
+            'X-Remote': true,
+          },
+          contentType: 'application/json',
+          method: 'PUT',
+        }).done(() => {
+          projectTitle.closest('form').removeClass('dirty');
+          setTimeout(window.location.reload(true), 2000);
```

Comment:
```suggestion
          $('.edit-project-board').closest('.board-column-header').find('.board-label').text(projectTitle.val());
```

Reply:
@jaqra didn't worked

Reply:
Morebspesicic: it updated but the popup form wont disapear

Reply:
```suggestion
          $(this).closest('.board-column-header').find('.board-label').text(projectTitle.val());
          $('.ui.modal').modal('hide');
```

Reply:
ok with this: backend changes title, form disapear, title **NOT** change no the UI and when you like to relaod a warning pops up say there is unstored contend